### DISCUSSION TITLE
position simulation

### DISCUSSION THREAD

**Discussion by urfausto** [osmandapp/OsmAnd#12812] [Q&A] [answered]
Hi, for the life of me i can't figure out what this feature (simulate your position) is for. can anybody explain it better to me?

**Comment by vshcherb** [comment, marked as answer]
This allows to test navigation without actual navigation

**Reply by urfausto** [reply]
> This allows to test navigation without actual navigation

ok but what is the difference with 'route simulation'?

**Reply by vshcherb** [reply]
Route simulation is only available for route, position simulation could simulate along any GPX (development plugin)

**Reply by urfausto** [reply]
ok thank you very much!

**Comment by urfausto** [comment]
anyone??
